specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2020-03-01/storagesync.json

```diff
@@ -278,6 +304,31 @@
               }
             }
           },
+          "202": {
+            "description": "Asynchronous Operation Status Location",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "Operation Status Location URI",
+                "type": "string"
+              },
+              "Location": {
+                "description": "Operation Status Location URI",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "Retry After",
+                "type": "string"
+              },
+              "x-ms-request-id": {
+                "description": "Request id",
+                "type": "string"
+              },
+              "x-ms-correlation-request-id": {
+                "description": "correlation request id",
+                "type": "string"
+              }
+            }
+          },
```

Comment:
And it seems not consistent within this json as well.

Does this one need a `"x-ms-long-running-operation": true` after `default`?